server/service/testing_utils.go

```diff
@@ -25,23 +25,24 @@ import (
 	"github.com/throttled/throttled/v2/store/memstore"
 )
 
-func newTestService(ds fleet.Datastore, rs fleet.QueryResultStore, lq fleet.LiveQueryStore, opts ...TestServerOpts) fleet.Service {
-	return newTestServiceWithConfig(ds, config.TestConfig(), rs, lq, opts...)
+func newTestService(t *testing.T, ds fleet.Datastore, rs fleet.QueryResultStore, lq fleet.LiveQueryStore, opts ...TestServerOpts) fleet.Service {
+	return newTestServiceWithConfig(t, ds, config.TestConfig(), rs, lq, opts...)
 }
 
-func newTestServiceWithConfig(ds fleet.Datastore, fleetConfig config.FleetConfig, rs fleet.QueryResultStore, lq fleet.LiveQueryStore, opts ...TestServerOpts) fleet.Service {
+func newTestServiceWithConfig(t *testing.T, ds fleet.Datastore, fleetConfig config.FleetConfig, rs fleet.QueryResultStore, lq fleet.LiveQueryStore, opts ...TestServerOpts) fleet.Service {
 	mailer := &mockMailService{SendEmailFn: func(e fleet.Email) error { return nil }}
 	license := &fleet.LicenseInfo{Tier: fleet.TierFree}
-	writer, _ := logging.NewFilesystemLogWriter(
+	writer, err := logging.NewFilesystemLogWriter(
 		fleetConfig.Filesystem.StatusLogFile,
 		kitlog.NewNopLogger(),
 		fleetConfig.Filesystem.EnableLogRotation,
 		fleetConfig.Filesystem.EnableLogCompression,
 	)
-	// See #1776
-	//if err != nil {
-	//	panic(err)
-	//}
+
+	if err != nil {
+		require.NoError(t, err)
+	}
```

Comment:
Nitpick: No need for an `if` clause, just:
```go
require.NoError(t, err)
```

Reply:
thanks!! I uncommented the code without thinking too much about it. Fix pushed.